web_patient/src/stores/PatientStore.ts

```diff
@@ -13,44 +19,36 @@ import {
     IValue,
     IValuesInventory,
 } from 'shared/types';
-import { IPatientService } from 'shared/patientService';
-import { IPromiseQueryState, PromiseQuery } from 'shared/promiseQuery';
-import { getLogger } from 'shared/logger';
+
 import { isScheduledForDay } from 'src/utils/schedule';
-// import { daysOfWeekValues } from 'shared/enums';
-import { getLoadAndLogQuery, onArrayConflict, onSingletonConflict } from 'shared/stores';
-import _ from 'lodash';
-import { compareDesc } from 'date-fns';
 
 export interface IPatientStore {
-    readonly taskItems: IScheduledActivity[];
-    readonly todayItems: IScheduledActivity[];
-    readonly assessmentsToComplete: IScheduledAssessment[];
-
-    readonly config: IPatientConfig;
-    readonly values: IValue[];
-    readonly valuesInventory: IValuesInventory;
-    readonly safetyPlan: ISafetyPlan;
-
     readonly activities: IActivity[];
-    readonly activitySchedules: IActivitySchedule[];
     readonly activityLogs: IActivityLog[];
+    readonly activitySchedules: IActivitySchedule[];
+    readonly assessmentsToComplete: IScheduledAssessment[];
     readonly assessmentLogs: IAssessmentLog[];
+    readonly config: IPatientConfig;
     readonly moodLogs: IMoodLog[];
+    readonly safetyPlan: ISafetyPlan;
+    readonly taskItems: IScheduledActivity[];
+    readonly todayItems: IScheduledActivity[];
+    readonly values: IValue[];
+    readonly valuesInventory: IValuesInventory;
 
     // UI states
-    loadPatientState: IPromiseQueryState;
-
-    loadActivitiesState: IPromiseQueryState;
-    loadActivityLogsState: IPromiseQueryState;
-    loadActivitySchedulesState: IPromiseQueryState;
-    loadAssessmentLogsState: IPromiseQueryState;
-    loadConfigState: IPromiseQueryState;
-    loadMoodLogsState: IPromiseQueryState;
-    loadSafetyPlanState: IPromiseQueryState;
-    loadScheduledActivitiesState: IPromiseQueryState;
-    loadValuesState: IPromiseQueryState;
-    loadValuesInventoryState: IPromiseQueryState;
+    readonly loadPatientState: IPromiseQueryState;
```

Comment:
I did not `edit` any code anywhere else except here. 

I added `readonly` for UI states in web_patient PatientStore to be consistent with web_registry PatientStore.